### PR TITLE
Login flow

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Use `<i>` instead of SVGs in static views
- Add static view for login
- Remove k8s api url from `Preferences`, as without authority data it wouldn't make sense anymore.
- Split backend into backend and npx app - no more ifNpx.

It turns out there are at least 3 types of kubeconfigs:
* With `user[*].token` - just use it as bearer token. Example: kk from Gardener cluster.
* With `user[*]['client-certificate-data']` and `user[*]['client-key-data']` - send it in custom headers and use on HTTPS agent in backend. Example: kk from k3d.
* Other - for example `exec`, which specifies OIDC login command (example: personal kk from Gardener). In that case we need to present 2 additional fields for issuer URL and client ID.

**BTW**

* You still can use enkode - just remember to add 'https://' to `server`.
* You can force login by removing `busola.init-params` from `sessionStorage`.

**Related issue(s)**

### CODE REVIEW

File: core/src/login.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+              id="error"
+              style="display: none"
+              class="fd-object-status fd-object-status--critical"
+            >
+              <i
+                class="fd-object-status__icon sap-icon--message-error"
+                role="presentation"
+              ></i>
+              <span class="fd-object-status__text"
+                >Error reading kubeconfig.</span
```
Comment (dariadomagala) [inline core/src/login.html:84]: Could we add an error message here?

**Review by dariadomagala — CHANGES_REQUESTED** [review]
On a cluster I get 
- ```Refused to load the script 'https://cdnjs.cloudflare.com/ajax/libs/js-yaml/4.0.0/js-yaml.js' because it violates the following Content Security Policy directive: "script-src 'self'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.``` in browser console
- ```error reading kubeconfig``` while adding new kubeconfig (the same kubeconfigs worked on localhost)

File: core/src/login.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+              </div>
+            </form>
+          </section>
+        </div>
+      </div>
+    </div>
+    <script
+      type="text/javascript"
+      charset="utf-8"
+      src="assets/login.js"
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply core/src/login.html:129]: Should we also move `static-views.js` then? I recognized `/assets` as folder for js files.

File: core/src/login.html (hunk 1 above)
Comment (Wawrzyn321) [reply core/src/login.html]: I didn't want to show the specific message from caught error, as it wouldn't give user any more information (like `Cannot read property '0' of undefined`).

File: core/src/login.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (dariadomagala) [reply core/src/login.html:132]: Ok, let's keep it here for now. We can think about restructuring it a bit with further refactors.

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**